terraform/ggroup-to-splunk/lambda.tf

```diff
@@ -5,24 +5,34 @@ data "aws_ssm_parameter" "google_credentials" {
 data "aws_ssm_parameter" "subject_email" {
   name = "/Google-Data-To-Splunk/subject_email"
 }
-  
+
+data "aws_ssm_parameter" "groups_scope" {
+  name = "/Google-Data-To-Splunk/groups_scope"
```

Comment:
Can you standardise on a `case`? IMHO it should all be `snake_case` but maybe we should discuss the possibility of `kebab-case`?

Reply:
agreed - we'll just change it so snake case